app/models/king.rb

```diff
@@ -1,4 +1,24 @@
 class King < Piece
-	def move
+
```

Comment:
additionally in a production app you'd want to make sure that the piece was not being told to move to the same location it was already on.  So start_point != end_point